src/urbit/sur/chess.hoon

```diff
@@ -210,8 +210,8 @@
     black=chess-player
   ::  %'0-1'
     result=(unit chess-result)
-  ::  a list of this round's moves
-    moves=(list chess-move)
+  ::  a list of this round's moves and corresponding fen
+    moves=(list [chess-move @t])
```

Comment:
We should probably declare a `chess-fen` type which just maps to `@t`, so that the type is self-documenting